### PR TITLE
DTS-41258

### PR DESCRIPTION
**Pull request by risshukl0**

### CODE REVIEW

**Review by spal-sapient — APPROVED** [review]

File: UI/src/app/dashboardv2/executive-v2/executive-v2.component.ts [hunk 1]
```diff
@@ -210,7 +210,8 @@ export class ExecutiveV2Component implements OnInit, OnDestroy {
   }
 
   setGlobalConfigData(globalConfig) {
-    this.configGlobalData = globalConfig[this.kanbanActivated ? 'kanban' : 'scrum'].filter((item) => (item.boardSlug?.toLowerCase() === this.selectedTab.toLowerCase()) || (item.boardName.toLowerCase() === this.selectedTab.toLowerCase().split('-').join(' ')))[0]?.kpis;
+    this.kanbanActivated = this.selectedtype.toLowerCase() === 'kanban' ? true : false;
```
Comment (prabasak23) [inline UI/src/app/dashboardv2/executive-v2/executive-v2.component.ts:213]: this assignment is not used anywhere, what is the purpose? @risshukl0

File: UI/src/app/dashboardv2/filter-v2/filter-new.component.ts [hunk 2]
```diff
@@ -813,7 +815,10 @@ export class FilterNewComponent implements OnInit, OnDestroy {
 
     this.filterApplyData['ids'] = [...new Set(event.map((item) => item.nodeId))];
     this.filterApplyData['selectedMap'][this.filterApplyData['label']] = [...new Set(event.map((item) => item.nodeId))];
-    let additionalFilterSelected = this.filterApplyData['label'] === this.squadLevel[0].hierarchyLevelId || this.filterApplyData['label'] === this.squadLevel[0].hierarchyLevelName ? true : false;
+    let additionalFilterSelected;
```
[the remark this answers is not in the document]
Comment (risshukl0) [reply UI/src/app/dashboardv2/filter-v2/filter-new.component.ts:818]: Restored the removed line @prabasak23

File: UI/src/app/dashboardv2/executive-v2/executive-v2.component.ts (hunk 1 above)
Comment (risshukl0) [reply UI/src/app/dashboardv2/executive-v2/executive-v2.component.ts:213]: It is there in html @prabasak23

**Review by prabasak23 — APPROVED** [review]
all looks ok. approving.

**Pull request merged**